ocs_ci/resiliency/storagecluster_component_failure.py

```
@@ -27,15 +28,27 @@ class StorageClusterComponentFailures:
         "MON_POD_FAILURES": "mon",
         "CEPHFS_POD_FAILURES": "cephfsplugin",
         "RBD_POD_FAILURES": "rbdplugin",
+        "RGW_POD_FAILURES": "rgw",
     }
 
     def __init__(self, failure_data):
         self.failure_data = failure_data
         self.disruptions = Disruptions()
         logger.info("Initialized StorageClusterComponentFailures")
 
+    def _rgw_pods_present(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_rgw_pods_present`

(ANN202)